packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -99,36 +99,19 @@ export default class CardAdoptionChain extends Component<Signature> {
   }
 
   @action
-  childFields(cardIndex: number): string[] {
-    const children = this.args.cardInheritanceChain.filter(
-      (_data, index) => index < cardIndex,
+  getInheritedFields(cardIndex: number, isGetParentField: boolean): string[] {
```

Comment:
The name `getInheritedFields` is confusing to me - is it really true that you get inherited fields?

Would it be clearer to have something like this?

```
getFields(cardIndex: number, from: "ancestors" | "successors");

getFields(cardIndex, "ancestors")
getFields(cardIndex, "successors")
```

Reply:
Good idea!